### DISCUSSION TITLE
Sourcemaps are not working in development

### DISCUSSION THREAD

**Discussion by Yankovsky** [tailwindlabs/headlessui#1954] [Help]
![image](https://user-images.githubusercontent.com/72094/197532731-5ca69d9a-93f7-49af-922e-9c9f8ecfd9fc.png)

The code is minified and the source maps don't work. Does anyone else experience the same problem?

**Comment by bitfactory-oussama-fadlaoui** [comment]
Will this be fixed? It's rather annoying when trying to debug exceptions

**Comment by Yankovsky** [comment]
What is the point of minifying the code in the library anyway? Every web app will bundle and minify all the sources in the end.
